projects/plugins/jetpack/CHANGELOG.md

```diff
@@ -8827,8 +8827,8 @@ Other bugfixes and enhancements at https://github.com/Automattic/jetpack/commits
 
 - Initial release
 
+[12.6]: https://wp.me/p1moTy-VLL
 [12.5]: https://wp.me/p1moTy-Vab
-[11.6]: https://wp.me/p1moTy-PLI
```

Comment:
I got sick of this always being out of order and tracked down the problem: https://github.com/Automattic/jetpack/pull/33217